sci-geosciences/qgis/qgis-9999.ebuild

```diff
@@ -50,6 +50,7 @@ COMMON_DEPEND="
 	>=dev-qt/qtsql-${QT_MIN_VER}:5
 	>=dev-qt/qtwidgets-${QT_MIN_VER}:5
 	>=dev-qt/qtxml-${QT_MIN_VER}:5
+	media-gfx/exiv2
```

Comment:
99% missing slot operator here, you'll need `:= to force rebuild on exiv2 subslot bump`

Reply:
`:=` added.